assets/css/main.css

```diff
@@ -29,14 +29,29 @@ section, p {
   clear: both;
 }
 
+button.primary {
+  height: 20px;
+  width: 72px;
+  font-size: 13px;
+  color: white;
+  border: none;
+  font-weight: 300;
+  border-radius: 3.5px;
+  background-image: linear-gradient(to bottom, #6cb3fa 10%, #067dff 100%);
```

Comment:
Also non-blocking but I have concern about making this blue. I wonder if there is a way to get the system styling for this? If not and maybe until we figure that out lets go with it.